authorization-common-api/src/main/java/com/sequenceiq/authorization/resource/AuthorizationResourceAction.java

```diff
@@ -45,6 +45,7 @@
     START_DATALAKE("datalake/startDatalake", AuthorizationResourceType.DATALAKE),
     STOP_DATALAKE("datalake/stopDatalake", AuthorizationResourceType.DATALAKE),
     UPGRADE_DATALAKE("datalake/upgradeDatalake", AuthorizationResourceType.DATALAKE),
+    ROTATE_CERT_DATALAKE("datalake/rotateCertDatalake", AuthorizationResourceType.DATALAKE),
```

Comment:
should we call it rotateTlsCertDatalake instead?

Reply:
renamed

Reply:
please update legacyRights.json also

Reply:
right, I will upload with the unit tests

Reply:
updated and unit test added